.git-blame-ignore-revs

```diff
@@ -19,3 +19,6 @@
 
 #
 ee3ecabf930faebc2e8c8c455c1a03dd591303d7
+
+## Reformat src and test to Apache NetBeans default ALT-SHIFT-F
+364b5432961e566c4f9b9eadbae566c0ed93db96
```

Comment:
the commit hash will probably change on merge

Reply:
no I use the default nebeans but the config checkstyle should use to be "muted" is google